src/tree/SubscriptionTreeItem.ts

```diff
@@ -27,14 +33,33 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
+
+        let treeItemPostgres: AzExtTreeItem[];
+        let treeItem: AzExtTreeItem[];
+
+        //Postgres
+        const clientPostgres: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
+        const accountsPostgres: ServerListResult = await clientPostgres.servers.list();
```

Comment:
Stay consistent with the `server` terminology. How about `pgServers` for this variable